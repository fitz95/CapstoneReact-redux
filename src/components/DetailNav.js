import React from 'react';
import { NavLink } from 'react-router-dom';
// import { FcSettings } from 'react-icons/fc';
import { MdArrowBackIos } from 'react-icons/md';
import styles from 'src/css/Navbar.module.css';

function DetailNav() {
  return (
    <header className={styles.header}>
      <NavLink to="/" className={styles.icondiv}>
        <MdArrowBackIos className={styles.icon2} />
      </NavLink>
      <span className={styles.navspan}>CRYPTO SAGE</span>
      {/* <nav>
        <ul className={styles.navItems}>
          <NavLink>
            <FcSettings className={styles.settingicon} />
          </NavLink>
        </ul>
      </nav> */}
    </header>
  );
}

export default DetailNav;
