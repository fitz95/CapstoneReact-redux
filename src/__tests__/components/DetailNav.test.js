import React from 'react';
import { BrowserRouter } from 'react-router-dom';
import renderer from 'react-test-renderer';
import DetailNav from 'src/components/DetailNav';

describe('Navbar', () => {
  it('renders correctly', () => {
    const tree = renderer
      .create(
        <BrowserRouter>
          <DetailNav />
        </BrowserRouter>,
      )
      .toJSON();
    expect(tree).toMatchSnapshot();
  });
});
