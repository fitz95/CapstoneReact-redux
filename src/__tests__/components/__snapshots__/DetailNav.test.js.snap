// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Navbar renders correctly 1`] = `
<header
  className="header"
>
  <a
    aria-current="page"
    className="icondiv active"
    href="/"
    onClick={[Function]}
  >
    <svg
      className="icon2"
      fill="currentColor"
      height="1em"
      stroke="currentColor"
      strokeWidth="0"
      style={
        Object {
          "color": undefined,
        }
      }
      viewBox="0 0 24 24"
      width="1em"
      xmlns="http://www.w3.org/2000/svg"
    >
      <path
        d="M0 0h24v24H0z"
        fill="none"
      />
      <path
        d="M11.67 3.87L9.9 2.1 0 12l9.9 9.9 1.77-1.77L3.54 12z"
      />
    </svg>
  </a>
  <span
    className="navspan"
  >
    CRYPTO SAGE
  </span>
</header>
`;
